src/modules/ce/components/ReferralStepCard.tsx

```diff
@@ -23,50 +34,83 @@ const ReferralStepCard: React.FC<Props> = ({ step }) => {
     opportunityId: string;
   };
   const { referral } = useReferralContext();
-  const { name, status } = step;
+  const { name, status, updatedBy, updatedAt } = step;
 
-  const action = useMemo(() => {
```

Comment:
instead of 3-dots menu, it should be the same action button as before - view or start